src/hooks/internal/useNavigation.ts

```diff
@@ -76,53 +132,108 @@ export const useNavigation: UseNavigationHook = (
             month: 'long' as const,
         }
 
+        const monthsData = getAvailableMonths(options.calendar)
+        const months = monthsData
```

Comment:
can we do all this mapping as part of the useNavigation, and just return the list of {value: integer, label: string} from there? That looks more logical for me

Reply:
I don't understand. Is this another suggestion instead of using JSON file?

Reply:
yeah sorry - the JSON suggestion supersedes this one.